WordPress/wordpress-playground
# Add Google Analytics events to Playground

## What is this PR doing?

This PR adds support for sending Google Analytics events.
Events are sent when the Playground website loads. There is a general _load_ event to track the number of loads, and separate events for each _step_ that send just the step name. 

## What problem is it solving?

We would like to better understand if people are using the features we build specifically what steps are used and how frequently.

## How is the problem addressed?

By sending step names to Google Analytics we will be able to track the total number of times a step has been used and compare it to the number of events created.

## Testing Instructions

- Checkout this branck
- Run `npm run dev`
- Add `console.log(event, data);` after this line https://github.com/WordPress/wordpress-playground/blob/05105b836e6122bbdc591861538aa0a78f3a21ca/packages/playground/website/src/lib/tracking.ts#L21 (Google Analytics only loads in production so we need to fake it locally)
- [Open Playground ](http://localhost:5400/website-server/?php=8.0&wp=6.3&plugin=hello-dolly&storage=none&php-extension-bundle=kitchen-sink)
- Confirm that the events were triggered in the browser console
